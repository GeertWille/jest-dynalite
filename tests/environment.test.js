const { default: Environment } = require("../dist/environment");

describe("environment", () => {
  it("should throw an error if config file could not be located", () => {
    expect(
      () => new Environment({ rootDir: "somebaddirectory" })
    ).toThrowErrorMatchingSnapshot();
  });
});
