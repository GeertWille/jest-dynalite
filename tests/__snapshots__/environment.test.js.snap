// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`environment should throw an error if config file could not be located 1`] = `
"
jest-dynalite could not find a config file in the jest <rootDir> (somebaddirectory).

If you didn't intend to be using this directory for the config, please specify a custom
directory: https://github.com/freshollie/jest-dynalite/#monorepoadvanced-setup

If you are already using a custom config directory, you should apply 'import \\"jest-dynalite/withDb\\"'
to your \\"setupFilesAfterEnv\\" instead of using the jest-dynalite preset or environment.

Please see https://github.com/freshollie/jest-dynalite/issues/6#issuecomment-654167172 for
more info.
      "
`;
